modules/Blockchain/Ethereum/migrations/2_total_migration.js

```diff
@@ -497,12 +497,24 @@ module.exports = async (deployer, network, accounts) => {
         // );
         break;
     case 'updateContract':
-        hub = await Hub.at('insert hub contract address here');
-        console.log(JSON.stringify(hub.address));
+        console.log(`Using wallet address: ${accounts[0]}`);
+        hub = await Hub.at('0x85720D4D09A52eD66656b28c68567E41AD45dD0e');
```

Comment:
Do we need to use hardcoded address here?

Reply:
This is hub contract address which shouldn't be changed